LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2017, katspaugh
```

Comment:
I'd say katspaugh and contributors. And look up when first commit of project was done and use that in copyright date, e.g. 2014-2017